helsinki/login/theme.properties

```diff
@@ -2,7 +2,7 @@ parent=keycloak
 import=common/keycloak
 locales=fi,sv,en
 # Inherit styles from parent and use custom css files
-styles=css/login.css css/hds.css css/shared.css css/hs-login.css
+styles=css/login.css css/hds.css?v=adad08bc css/shared.css?v=05a34181 css/hs-login.css?v=12e8db5e
```

Comment:
The `login.css` is not part of this theme, it comes from the Keycloak base theme. It doesn't need this hash treatment because whenever the base theme changes, Keycloak version changes and that changes the "hash part" in the URL path: https<span></span>://tunnistus.dev.hel.ninja/auth/resources/<b>xmykb</b>/login/helsinki/css/login.css.